src/components/Button/index.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import Button from '.';
+
+const buttonTestId = 'login-form__button-submit';
+
+describe('Button', () => {
+  it('renders the button', () => {
+    render(<Button disabled={false} name={'Sign in'} className="sign-in-btn" data-test-id={buttonTestId}></Button>);
```

Comment:
It's part of our convention to self-close tags :D 
https://nimblehq.co/compass/development/code-conventions/javascript/react/#tags

```suggestion
    render(<Button disabled={false} name={'Sign in'} className="sign-in-btn" data-test-id={buttonTestId} />);
```

Reply:
Could you also check the other tests as well 👍

Reply:
Fixed in 60500ea60936d447aa25f8f3bf998a7ece9ab7a7 😀